packages/kotti-ui/source/kotti-drawer/KtDrawer.vue

```diff
@@ -1,203 +1,195 @@
-		},
-		onOutsideDrawerClick() {
-			if (this.closeOutside) {
-				this.$emit('close')
-			}
-		},
+			},
+			Yoco,
+		}
 	},
-}
+})
 </script>
 
 <style lang="scss">
+:root {
+	--default-drawer-width: 18.8rem;
+	--wide-drawer-width: 26.8rem;
+}
 @import '../kotti-style/_variables.scss';
 
 // Drawer with Vue Transition
-$default-drawer-width: 18.8rem;
-$wide-drawer-width: 26.8rem;
-
-.drawer-mask {
-	position: fixed;
-	top: 0;
-	left: 0;
-	z-index: $zindex-4;
-	display: flex;
-	width: 100%;
-	height: 100%;
-	background-color: rgba(0, 0, 0, 0.5);
-	transition: opacity 0.5s ease;
-}
 
-.drawer-handle {
-	position: absolute;
-	top: 50%;
-	left: var(--unit-1);
-	display: block;
-	width: 1.5rem;
-	height: 1.5rem;
-	margin-top: -0.725rem;
-	line-height: 1.5rem;
-	color: var(--icon-02);
-	cursor: pointer;
-
-	&:hover {
-		color: var(--interactive-03);
+.kt-drawer {
+	&__mask {
+		position: fixed;
+		top: 0;
+		left: 0;
+		z-index: $zindex-4;
+		display: flex;
+		width: 100%;
+		height: 100%;
+		background-color: rgba(0, 0, 0, 0.5);
+		transition: opacity 0.5s ease;
 	}
-}
 
-.drawer-container {
-	position: absolute;
-	top: 0;
-	right: 0;
-	display: flex;
-	flex-direction: column;
-	width: $default-drawer-width;
-	height: 100%;
-	padding: var(--unit-6);
-	padding-left: var(--unit-8);
-	overflow-y: auto;
-	background-color: var(--ui-background);
-	box-shadow: 0 2px 8px rgba(0, 0, 0, 0.33);
-	transition: all 0.3s ease;
-
-	&.drawer-expand {
-		width: 50%;
+	&__container {
+		position: absolute;
+		top: 0;
+		right: 0;
+		display: flex;
+		flex-direction: column;
+		width: var(--default-drawer-width);
+		height: 100%;
+		padding: var(--unit-6);
+		padding-left: var(--unit-8);
+		overflow-y: auto;
+		background-color: var(--ui-background);
+		box-shadow: 0 2px 8px rgba(0, 0, 0, 0.33);
+		transition: all 0.3s ease;
+
+		&--is-expanded {
+			width: 50%;
+		}
+
+		&--is-wide {
+			width: var(--wide-drawer-width);
+
+			&.kt-drawer__container--is-expanded {
+				width: 75%;
+			}
+		}
 	}
-}
 
-.drawer-container.drawer-wide {
-	width: $wide-drawer-width;
+	&__handle {
+		position: absolute;
+		top: 50%;
+		left: var(--unit-1);
+		display: block;
+		width: 1.5rem;
+		height: 1.5rem;
+		margin-top: -0.725rem;
+		line-height: 1.5rem;
+		color: var(--icon-02);
+		cursor: pointer;
+
+		&:hover {
+			color: var(--interactive-03);
+		}
+	}
 
-	&.drawer-expand {
-		width: 75%;
+	&__header {
+		flex: 0 0 auto;
 	}
-}
 
-.drawer-header {
-	flex: 0 0 auto;
-}
+	&__body {
+		flex: 1 1 auto;
+		padding: 0 0.8rem;
+		margin: 0 -0.8rem 0.8rem -0.8rem;
+		overflow-y: auto;
+	}
 
-.drawer-body {
-	flex: 1 1 auto;
-	padding: 0 0.8rem;
-	margin: 0 -0.8rem 0.8rem -0.8rem;
-	overflow-y: auto;
-}
+	&__footer {
+		flex: 0 0 auto;
+		align-self: flex-end;
+		width: 100%;
+	}
 
-.drawer-footer {
-	flex: 0 0 auto;
-	align-self: flex-end;
-	width: 100%;
-}
+	/* Vue Transition classes */
 
-.drawer-enter {
-	opacity: 0;
-	transition: opacity 0.5s;
-}
+	&-enter {
```

Comment:
is this why we can't add `scoped` to the style sheet

can we `::v-deep` on each of the sub-classes (won't work with scss syntax) instead?

Reply:
It’s about the `:root`. I changed it to only make that part non-scoped